libarchive/PKGBUILD

```diff
@@ -12,7 +12,7 @@ depends=('gcc-libs' 'libbz2' 'libiconv' 'libexpat' 'liblzma' 'liblz4' 'liblzo2'
 makedepends=('libbz2-devel' 'libiconv-devel' 'libexpat-devel' 'liblzma-devel' 'liblz4-devel' 'liblzo2-devel' 'libnettle-devel' 'libxml2-devel' 'zlib-devel')
 options=('!strip' 'debug' 'libtool')
 # provides=('libarchive.so')
-source=("https://libarchive.org/downloads/$pkgname-$pkgver.tar.gz"
+source=("https://github.com/libarchive/libarchive/archive/v${pkgver}.tar.gz"
```

Comment:
That's an unrelated change, I would back it out.

Reply:
I changed it because the `https://libarchive.org/downloads` url is 404.